src/ui/__tests__/__snapshots__/bot.spec.ts.snap

```diff
@@ -14,6 +14,11 @@ exports[`DOM behaviors user reply 1`] = `
         <div
           class="yvebot-message"
         >
+          <div
```

Comment:
I prefer to keep the sender outside of message content.
```html
<li class="yvebot-thread ...">
  <div class="yvebot-bubble">
    <div class="yvebot-sender"></div>
    <div class="yvebot-message"></div>
  </div>
</li>
``` 
You have to move the css of `yvebot-message` to `yvebot-bubble` in themed example